opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -1285,6 +1413,10 @@ namespace detail {
         const V dxvar = active_[Gas] ? subset(dx, Span(nc, 1, varstart)): null;
         varstart += dxvar.size();
 
+        // Extract well parts np phase rates + bhp
+        const V dwells = subset(dx, Span((np+1)*nw, 1, varstart));
+        varstart += dwells.size();
+
         const V dqs = subset(dx, Span(np*nw, 1, varstart));
```

Comment:
The extraction of dqs and dbhp should be removed: it is now wrong (since varstart is now at the end) and they are not required in this method any more.